### ISSUE TITLE
Output to file

### ISSUE THREAD

**Issue by thibautjombart** [Hackout3/epimatch#9]
Comment from Peter White (PHE): it looks awesome! But it would be very useful to be able to output results to a file for further inspection and/or sharing and/or analysis.

**Comment by daattali** [comment]
Sounds like a good idea indeed

I initially thought I would just add it to the shiny interface, but I actually think it would be better it there was an independent function that would take the results from epimatch and write them to a file. Then it could either be called from CLI or from shiny.

**Comment by thibautjombart** [comment]
Makes total sense!

**Comment by zkamvar** [comment]
Makes sense to me! Given the heterogeneous nature of the matches across spreadsheets, what kind of output files would be desirable? 

I'm imagining a series of files, ordered from nearest to farthest match with the matching columns within (excluding non-matching columns so they can be merged into the same table). Does this sound like something that could be useful, or would it need all of the columns (including the ones not used for matching)?

**Comment by zkamvar** [comment]
Note: I've started to work on this in https://github.com/Hackout3/epimatch/tree/write-data
